iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -41,13 +41,13 @@
   <ItemGroup>
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.6.0" />
     <PackageReference Include="DotNetty.Handlers" Version="0.6.0" />
-    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.3.7" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.2" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="$(RootDir)\shared\src\Microsoft.Azure.Devices.Shared.csproj" />
+    <ProjectReference Include="..\..\..\..\..\azure-amqp\Microsoft.Azure.Amqp\Microsoft.Azure.Amqp.csproj" />
```

Comment:
Actually we will use the NuGet package instead of local source.

---
In reply to: [251634768](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r251634768) [](ancestors = 251634768)